src/components/RoomNameInput/types.ts

```diff
@@ -0,0 +1,22 @@
+import type {ForwardedRef} from 'react';
+import type {NativeSyntheticEvent, ReturnKeyTypeOptions, TextInputFocusEventData, TextInputSubmitEditingEventData} from 'react-native';
+import type {MaybePhraseKey} from '@libs/Localize';
+import type {BaseTextInputRef} from '@src/components/TextInput/BaseTextInput/types';
+
+type RoomNameInputProps = {
+    value?: string;
+    disabled?: boolean;
+    errorText?: MaybePhraseKey;
+    onChangeText?: (value: string) => void;
+    forwardedRef?: ForwardedRef<BaseTextInputRef>;
+    onSubmitEditing?: (event: NativeSyntheticEvent<TextInputSubmitEditingEventData>) => void;
+    onInputChange?: (value: string) => void;
+    returnKeyType?: ReturnKeyTypeOptions;
+    inputID?: string;
+    onBlur?: (event: NativeSyntheticEvent<TextInputFocusEventData>) => void;
+    autoFocus?: boolean;
+    shouldDelayFocus?: boolean;
+    isFocused?: boolean;
```

Comment:
`isFocused` prop was required before, let's track down the usage of `RoomNameInput` and make some of these props required too